src/odemis/acq/move.py

```diff
@@ -77,32 +125,30 @@ def check_axes(pos):
         return None
 
 
-def _isInRange(stage, axis):
+def _isInRange(current_pos, active_range, axis):
```

Comment:
I think that for a couple of functions you forgot to add some input values to the docstring